QGCExternalLibs.pri

```diff
@@ -244,21 +244,35 @@ contains (DEFINES, DISABLE_AIRMAP) {
         }
     } else:LinuxBuild {
         #-- Download and install platform-sdk libs and headers iff they're not already in the build directory
+        isEmpty(AIRMAP_PLATFORM_SDK_FILEPATH){
```

Comment:
Ok. I was a bit paranoid when I thought about the undefined case. Don't know that much about QMake so far, so I decided to go this way instead of making anything unstable.
Will be corrected.

Reply:
please see my comment below